### PR TITLE
chore: bump version for new release

### PR DESCRIPTION
**Pull request by iamsobanjaved**
**TL;DR -** Bumping version for new release

**What changed?**

- The previous release is broken, so a revert is being made and creating a new release.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by awais786 — APPROVED** [review]

**Comment by pomegranited** [comment]
Closing obsolete PR.
